deploy/charts/values.yaml

```
@@ -94,15 +94,35 @@ kube-prometheus-stack:
 
     config:
       global:
+        ## smtp host which would relay emails to the recepient eg: smtp.office.com:587.
+        smtp_smarthost: ''
+        ## from address on the email notification.
+        smtp_from: ''
+        ## username to authenticate with smtp host
+        smtp_auth_username: ''
+        ## pawwrod for the username.
+        smtp_auth_password: ''
```

Why are we adding comments here? Isn't this well documented with alert manager anyway? So perhaps we just need to point them that way?

```suggestion
        smtp_smarthost:
        ## from address on the email notification.
        smtp_from:
        ## username to authenticate with smtp host
        smtp_auth_username:
        ## password for the username.
        smtp_auth_password:
```